bcda/auth/plugin/alpha_test.go

```diff
@@ -95,9 +96,10 @@ func (s *AlphaAuthPluginTestSuite) TestValidateAccessToken() {
 }
 
 func (s *AlphaAuthPluginTestSuite) TestDecodeAccessToken() {
-	t, err := s.p.DecodeAccessToken("")
+	ts, _ := auth.InitAuthBackend().GenerateTokenString("504", "911")
```

Comment:
You should use UUIDs as params here, since that's what we use in the tokens we generate. You can use uuid.NewRandom().String(), or you can consult the test for GenerateTokenString() in auth/backend_test.go for fixture data UUIDs. This will important with respect to parsing claims and validation.